sdk/servicebus/azure-messaging-servicebus/docs/Cross-Entity-Transaction.md

```diff
@@ -0,0 +1,33 @@
+    2. All actions performed from the transactional sender/receiver will be associated with the transactionId of the transaction. Non-transactional actions should not be performed with transactional sender/receiver. 
+    3. The existing "send via" entity path parameter should not be set by the user anymore. This value will be inferred by the service side instead. Details will be shown below. 
+  
+3. Deciding the "via" sender: 
+    1. When the first transactional sender/receiver is created with an associated transaction, the target entity of this sender/receiver will be chosen by ServiceBus server side as the " via" entity implicitly. 
+    2. The "via" entity must be a top level entity (queue or topic). If the target entity is a subscription, then the corresponding topic will be chosen to be the "send via" entity. 
+    3. All subsequent transactional senders under the same transaction will "send via" the "via" entity chosen above. 
+  
+4. Constraints of this new model are same as existing model except c.: 
+    1. Either transactional sender or transactional receiver can be the first link to be created with a specific transaction and have its target entity chosen as the "via" entity for that transaction. All subsequent links created with that transaction must be senders (otherwise we are doing "receive via"), this is the same as today. Violations will result in exceptions thrown from server. 
+    2. As an exception to rule a. above, transactional receivers on the same transaction are still allowed after the "via" entity is established as long as its top level entity is the same as the "via" entity. This scenario is acceptable because it will no longer be a cross-entity transaction. 
+    3. All transactional actions performed using the transactional senders/receivers described above will be transactional with the same transactionId and sending via the "via" entity whenever applicable. There is no option to turn this off. 
+
+5. AMQP implementation changes to support this new model: 
+    1. The current AMQP model has 1 to 1 mapping between all AMQP links and sessions, and each sender/receiver/transaction will allocate its individual AMQP session+link upon creation. The proposed model will have each new transaction create their own AMQP session and coordinator link, and all transactional senders/receivers associated with that transaction will only allocate their AMQP links, but share the same AMQP session with the associated transaction. (Please see diagrams below). 
+  
+  
+5. ViaPartitionKey: 
+    1. ViaPartitionKey will remain the same as now with no changes at all. It is still required by partitioned entities during a transaction. 
+  
+  
+ <img width="899" alt="Screen Shot 2021-08-05 at 10 21 22 AM" src="https://user-images.githubusercontent.com/1471612/128393795-68af7fd2-ec5a-4cf7-9587-3d08e1e31987.png">
```

Comment:
I would add this into our repository next to the MD file and link it. In the case that the content disappears or others can change it.